src/ui/src/builder/settings/BuilderSettingsProperties.vue

```diff
@@ -240,6 +239,14 @@ function handleExpand(fieldKey: string) {
 function handleShrink(fieldKey: string) {
 	expandedFields.value.delete(fieldKey);
 }
+
+interface Props {
+	groupLevel: 0 | 1;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+	groupLevel: 0,
+});
```

Comment:
It's better to use object notation to define props
```suggestion
const props = defineProps({
	groupLevel: { type: Number as PropType<0 | 1>, required: false, default: 0 },
});
```